reliability/setup.yml

```diff
@@ -0,0 +1,166 @@
+---
```

Comment:
wait -- we have implemented a register tools role in pbench.  please, please please use that instead so we can all rally around a single tool.  @chaitanyaenr can you please help here?